main/lua-xml/APKBUILD

```diff
@@ -1,11 +1,11 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 _luaversions="5.1 5.2"
-pkgname=lua-xml
+pkgname=luaxml
```

Comment:
All Lua libraries are named as `lua-<name>` or lua<lver>-<name>`, where `<lver>` is Lua version (e.g. lua-xml, lua5.1-xml, …).